LLAdd/LLAdd1.trig

```diff
@@ -20,6 +20,8 @@
     dcterms:format "application/trig" ;
     prov:asDerivedFrom <https://www.vanhamel.nl/codecs/Atkinson_1880> .
 
+# p.378a53
```

Comment:
I have done some very small investigation into how to cite journal articles/books in RDF.  I will have a more definitive look later and come back with something so we can ingest this information property.